{"name": "test_auditTrailUnitAdd", "status": "broken", "statusDetails": {"message": "UnboundLocalError: local variable 'count' referenced before assignment", "trace": "def test_auditTrailUnitAdd():\n>       TestCoverageUnit.auditTrailUnitAdd(driver,oldName,oldDescription,\"No\")\n\nTestCaseUnit.py:54: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nTestCoverage\\TestCoverageUnit.py:369: in auditTrailUnitAdd\n    beforeCount=TestCoverageAudittrail.auditTrailRecordCount(driver)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"641ed8a5d34bee30fa44fe59fef88f89\")>\n\n    def auditTrailRecordCount(driver):\n    \n        ScreenNavigate.auditTrailPreCondition(driver)\n    \n        try:\n            countText= driver.find_element(By.XPATH,\"(//*[@class='k-pager-info k-label'])[2]\").text\n    \n            individualText = countText.split(' ')\n    \n            count = individualText[4]\n    \n            logger.info(\"The count is fetched, it shows {} record in the grid\".format(count))\n    \n            count = int(count)\n    \n        except:\n    \n            try:\n                time.sleep(3)\n    \n                countText = driver.find_element(By.XPATH, \"(//*[@class='k-pager-info k-label'])[2]\").text\n    \n                individualText = countText.split(' ')\n    \n                count = individualText[4]\n    \n                logger.info(\"The count is fetched, it shows {} record in the grid\".format(count))\n    \n                count = int(count)\n    \n            except:\n    \n                logger.error(\"Unable to get the count in the Audit trail screen\")\n    \n    \n        ScreenNavigate.auditTrailPostCondition(driver)\n    \n>       return count\nE       UnboundLocalError: local variable 'count' referenced before assignment\n\nTestCoverage\\TestCoverageAudittrail.py:52: UnboundLocalError"}, "attachments": [{"name": "log", "source": "4ef12703-ed41-4f42-8bf7-355b892d4752-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "e0d408a2-c373-4a13-abbe-a961f79e82e4-attachment.txt", "type": "text/plain"}], "start": 1650649626017, "stop": 1650649751328, "uuid": "21be31bb-625d-446f-a5bc-cc727a3fdfdc", "historyId": "8e44131e7c2b648eb351c090d10dcad7", "testCaseId": "f5a9c943acd3daab2b8e9ddb3034b3e7", "fullName": "TestCaseUnit#test_auditTrailUnitAdd", "labels": [{"name": "suite", "value": "TestCaseUnit"}, {"name": "host", "value": "AGL78"}, {"name": "thread", "value": "31084-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCaseUnit"}]}